core/src/HIP/Kokkos_HIP_KernelLaunch.hpp

```diff
@@ -542,8 +540,13 @@ struct HIPParallelLaunchKernelInvoker<DriverType, LaunchBounds,
          get_kernel_func())<<<grid, block, shmem, hip_instance->m_stream>>>();
 
     // Record an event that says when the constant buffer can be reused
-    KOKKOS_IMPL_HIP_SAFE_CALL(hip_instance->hip_event_record_wrapper(
-        HIPInternal::constantMemReusable[hip_device]));
+    KOKKOS_IMPL_HIP_SAFE_CALL(hipLaunchHostFunc(
```

Comment:
It seems that `hipLaunchhostFunc` does not exist for `HIP 5.3`.

Reply:
For `HIP` `5.3` or below, we'll still have to use `hipStreamAddCallback`.

Reply:
OK.  Note that the ROCm documentation still marks `hipLaunchhostFunc` as "beta" in the latest version (6.4)